conans/test/integration/package_id/test_validate.py

```diff
@@ -64,17 +64,43 @@ def package_id(self):
         self.assertIn("pkg/0.1: Package 'cb054d0b3e1ca595dc66bc2339d40f1f8f04ab31' created",
                       client.out)
 
-        client.run("create . pkg/0.1@ -s os=Windows")
-        self.assertIn("pkg/0.1: Main binary package 'INVALID' missing. "
-                      "Using compatible package 'cb054d0b3e1ca595dc66bc2339d40f1f8f04ab31'",
-                      client.out)
-        self.assertIn("pkg/0.1:cb054d0b3e1ca595dc66bc2339d40f1f8f04ab31 - Cache", client.out)
+        client.run("create . pkg/0.1@ -s os=Windows", assert_error=True)
+        self.assertIn("pkg/0.1: Invalid: Windows not supported", client.out)
+        self.assertIn("pkg/0.1:INVALID - Invalid", client.out)
         client.run("info pkg/0.1@ -s os=Windows")
-        self.assertIn("pkg/0.1: Main binary package 'INVALID' missing. "
-                      "Using compatible package 'cb054d0b3e1ca595dc66bc2339d40f1f8f04ab31'",
+        self.assertIn("pkg/0.1: Main binary package '3475bd55b91ae904ac96fde0f106a136ab951a5e' "
+                      "missing. Using compatible package 'cb054d0b3e1ca595dc66bc2339d40f1f8f04ab31'",
```

Comment:
Why the info says it use the compatible but the create failed?

Reply:
I need some clarification about this one as well

Reply:
The ``conan info`` command doesn't build, it is a proxy for ``install`` not for create. If we did ``info --dry-build=pkg``, it would not fall back and report ``pkg/0.1 ID: INVALID``